packages/common/driver-definitions/src/urlResolver.ts

```diff
@@ -4,36 +4,80 @@
  */
 import { IRequest } from "@fluidframework/core-interfaces";
 
+/**
+ * TODO
+ */
```

Comment:
A bit of info on this one: 
`IWebResolvedUrl` [is gone](https://github.com/microsoft/FluidFramework/blob/next/BREAKING.md#IResolvedUrl-equivalent-to-IFluidResolvedUrl) and [`IFluidResolvedUrl` is now deprecated](https://github.com/microsoft/FluidFramework/pull/14897), so we're moving towards a world where the only one left is `IResolvedUrl`.